File: 1 - Multiples of 3 or 5/go/main.go
package main

func solve() int {

	var sum int = 0

	for i := 0; i < 1000; i++ {
		if i%3 == 0 || i%5 == 0 {
			sum += i

		}
	}
	return sum
}

func main() {
	var solution int = solve()
	println(solution)

}

File: 1 - Multiples of 3 or 5/rust/main.rs
fn solve() -> i32 {
    let mut sum : i32 = 0;
    for i in 0 .. 1000 {
        if i % 3 == 0 || i % 5 == 0 {
            sum += i;
        }
    }
    return sum;
}

fn main() {
    println!("{}", solve());
}
